rules/import-index.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const isImportingIndex = m => m.endsWith('/') || m.endsWith('index.js') || m.endsWith('index');
```

Comment:
@jfmengels I guess you didn't mean `index` and `index.js`? Regarding the case `@foo/index.js`, does it mean the package is called `@foo/index.js` and actually you are importing `@foo/index.js/index.js`?